docs/gui.rst

```diff
@@ -45,25 +46,25 @@ Create a window
                 gui.show(f'{frame:06d}.png')
 
 
-Paint a window
---------------
+Paint on a window
+-----------------
 
 
 .. function:: gui.set_image(img)
 
     :parameter gui: (GUI) the window object
     :parameter img: (np.array or Tensor) tensor containing the image, see notes below
 
-    Set a image to display on the window.
+    Set an image to display on the window.
 
-    The pixel, ``i`` from bottom to up, ``j`` from left to right, is set to the value of ``img[i, j]``.
+    The window pixels, ``i`` from left to right, ``j`` from bottom to top, are set to the values of ``img[i, j]``.
```

Comment:
Thank for fixing the typo.. but I think `img[i, j]` is one pixel (one value), instead of pixels (values).
```suggestion
    The window pixel at ``i`` from left to right and ``j`` from bottom to top, is set to the value of ``img[i, j]``.
```

Reply:
I'd say ''The image pixels are set from the values of ``img[i,j]``, where ``i`` indicates the horizontal coordinates from left to right while ``j`` the vertical coordinates from bottom to top.